garage/envs/mujoco/mujoco_env.py

```diff
@@ -230,3 +230,9 @@ def print_stats(self):
 
     def action_from_key(self, key):
         raise NotImplementedError
+
+    def _env_setup(self, initial_qpos):
```

Comment:
a private API which is commonly called by subclasses is a bad smell. I think an initial_qpos field in the constructor of MujocoEnv could satisfy the same purpose, right?

Reply:
Yeah, the init_qpos is acquired from the simulation initial qpos in __init__() but I actually want to have initial_qpos set by users. This function is used mostly for robotics. I will modify it as part of the mujoco_env initialization and have a parameter with default None. Does it sound good?